.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "cSpell.words": [
+    "cref",
+    "otel",
+    "unvalidated"
+  ],
+  "dotnet-test-explorer.testProjectPath": "test/**/*.Tests.csproj"
+}
```

Comment:
Personally, I am not using any of these extensions and I think that these configs would get outdated.

Reply:
I disagree those particular configs would get outdated - they'd be maintained as the project grows. Being able to catch spelling errors is pretty helpful and used to be something FxCop would do. See, for example, [this `jeager-host` typo](https://github.com/open-telemetry/opentelemetry-dotnet/blob/8b8b9348aafeb4629b6a8261e327dccac33e9a0d/test/OpenTelemetry.Exporter.Jaeger.Tests/JaegerExporterOptionsTests.cs#L57).

And I'm pretty sure the test projects have a pattern that wouldn't change, but I feel much less strongly about whether folks want the .NET test explorer.

Regardless, I don't find value in arguing it. It seems like it'd be valuable to help folks - at least VS Code folks - avoid simply typos, but if it's not interesting I can remove it.

Reply:
It does appear [there's a spelling check that runs during the `sanitycheck` build stage](https://github.com/open-telemetry/opentelemetry-dotnet/blob/8b8b9348aafeb4629b6a8261e327dccac33e9a0d/.github/workflows/sanitycheck.yml#L23) but I'm not sure where its dictionary is or why it's not erring out things like `jeager`.

Reply:
Entirety of `settings.json` removed.